sup3r/preprocessing/data_handling/base.py

```diff
@@ -1269,14 +1200,12 @@ def serial_data_fill(self, shifted_time_chunks):
         for t, ts in enumerate(shifted_time_chunks):
             for _, f in enumerate(self.noncached_features):
                 f_index = self.features.index(f)
-                self.data_fill(t, ts, f_index, f)
-            interval = int(np.ceil(len(shifted_time_chunks) / 10))
-            if t % interval == 0:
-                logger.info(f'Added {t + 1} of {len(shifted_time_chunks)} '
-                            'chunks to final data array')
+                self._data_fill(t, ts, f_index, f)
+            logger.info(f'Added {t + 1} of {len(shifted_time_chunks)} '
+                        'chunks to final data array')
             self._raw_data.pop(t)
 
-    def parallel_data_fill(self, shifted_time_chunks, max_workers=None):
+    def data_fill(self, shifted_time_chunks, max_workers=None):
```

Comment:
Not a huge fan of this naming convention. I always find it confusing when there is a `.method` and a `._method`. I think you're trying to hide the serial method from the "public" view but it makes the developer's job more confusing. I'd suggest `_single_data_fill()` and `_data_fill()`